dali/pipeline/operator/op_schema.cc

```diff
@@ -111,6 +111,33 @@ std::map<std::string, DefaultedArgumentDef> OpSchema::GetOptionalArguments() con
   return ret;
 }
 
+DLL_PUBLIC bool OpSchema::IsDeprecatedArg(const std::string &arg_name) const {
+  if (deprecated_arguments_.find(arg_name) != deprecated_arguments_.end())
+    return true;
+  for (const auto &parent_name : parents_) {
+    const OpSchema &parent = SchemaRegistry::GetSchema(parent_name);
+    if (parent.IsDeprecatedArg(arg_name))
+      return true;
+  }
+  return false;
+}
+
+DLL_PUBLIC const DeprecatedArgDef &OpSchema::DeprecatedArgMeta(const std::string &arg_name) const {
+  auto it = deprecated_arguments_.find(arg_name);
+  if (it != deprecated_arguments_.end()) {
+    return it->second;
+  }
+  for (const auto &parent_name : parents_) {
+    const OpSchema &parent = SchemaRegistry::GetSchema(parent_name);
+    if (parent.IsDeprecatedArg(arg_name))
+      return parent.DeprecatedArgMeta(arg_name);
+  }
+  DALI_ENFORCE(IsDeprecatedArg(arg_name),
```

Comment:
Either enforce or throw.

Reply:
I think the enforce is redundant. If you reached here you know that IsDeprecatedArg would return false.

Reply:
done